ext/native-decls/GetVehicleGearRatio.md

```diff
@@ -0,0 +1,38 @@
+---
+ns: CFX
+apiset: client
+---
+## GET_VEHICLE_GEAR_RATIO
+
+```c
+float GET_VEHICLE_GEAR_RATIO(Vehicle vehicle, int gear);
+```
+Gets vehicles gear ratio on choosen gear.
+
+## Parameters
+* **vehicle**: The vehicle handle.
+* **gear**: The vehicles gear you want to get.
+
+## Examples
+```lua
+local Vehicle = GetVehiclePedIsIn(PlayerPedId(-1))
+local currentGear = GetVehicleCurrentGear(Vehicle)
+
+print(GetVehicleGearRatio(Vehicle, currentGear)) -- will print current vehicle gear to console
+```
+
+```js
+const Vehicle = GetVehiclePedIsIn(PlayerPedId(-1));
```

Comment:
Please maintain consistent usage of the same codestyle throughout, if `lowerCamelCase` is used for variables in JS examples, keep using it across all JS examples.